app/services/hyrax/actor_factory.rb

```diff
@@ -9,9 +9,10 @@ def self.stack_actors(curation_concern)
        Hyrax::Actors::AttachFilesActor,
        Hyrax::Actors::ApplyOrderActor,
        Hyrax::Actors::InterpretVisibilityActor,
+       DefaultAdminSetActor,
+       Hyrax::Actors::InitializeWorkflowActor,
```

Comment:
Why is Initialize moving up?  Don't we need to save the model before we activate the workflow?